man/systemd.network.xml

```diff
@@ -317,6 +317,47 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+      <title>[MPTCP] Section Options</title>
+      <para>The [MPTCP] section accepts the following keys. Specify several [MPTCP] sections to configure
+      several MPTCPs. MPTCP is a transport protocol built on top of TCP that allows TCP connections to use
+      multiple paths to maximize resource usage and increase redundancy.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Id=</varname></term>
+          <listitem>
+            <para>Specifies a a unique numeric identifier for the given endpoint. Takes an unsigned integer in the range 0..255.</para>
```

Comment:
"a a" is a typo

(and i guess you can drop the word "unsigned" from this line, after all the range you specify already makes that clear enough, but this doesn't matter)